kpi/views/Automated_KPI_Forecast_Confidence_Intervals.view.lkml

```diff
@@ -0,0 +1,154 @@
+include: "//looker-hub/kpi/views/*"
+
+view: +automated_kpi_confidence_intervals {
+
+  dimension: asofdate {
+    label: "Last Forecasted Date"
+  }
+
+  dimension: date {
+    hidden: yes
+    sql: ${TABLE}.date ;;
+    }
+
+  dimension_group: submission {
+    sql: CAST(${TABLE}.date as DATE) ;;
+    type: time
+    timeframes: [
+      date,
+      month,
+      month_name,
+      quarter,
+      year,
+    ]
+    convert_tz: no
+    datatype: date
+  }
+
+  dimension: forecast_date {
+    label: "Forecast Run Date"
+    }
+
+  dimension: value {
+    hidden:  yes
+    sql: ${TABLE}.value ;;
+  }
+
+  measure: Estimated_Value {
+    type: number
+    description: "Forecasted if after forecast date, observed if before."
```

Comment:
which value is this using if its after the forecast date? 50th percentile? Also, maybe re-phrase: "Forecast Date" to "Forecast Run Date" to match the label

Reply:
Fixed the label. The value isn't the 50th percentile, it's the average of the uncertainty samples. (I've also added this to the label)